tscv/_split.py

```diff
@@ -10,8 +10,9 @@
 import numbers
 from math import modf
 from abc import ABCMeta, abstractmethod
-from itertools import chain
+from itertools import chain, combinations
 from inspect import signature
+from scipy.special import comb
```

Comment:
Please add `scipy` in https://github.com/WenjieZ/TSCV/blob/2abbc3d34151555ea12c9abb13d7a76ead5a20a8/setup.py#L55